packages/react/src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,56 @@
+import type { HTMLAttributes } from 'react';
+import React, { forwardRef } from 'react';
+import cn from 'classnames';
+import { CheckmarkIcon } from '@navikt/aksel-icons';
+
+import type { OverridableComponent } from '../../types/OverridableComponent';
+
+import classes from './Chip.module.css';
+import { ChipButton, type ChipButtonProps } from './_ChipButton';
+
+export type ChipProps = ChipButtonProps & {
+  /**
+   * Condition if the check mark icon should be displayed
+   */
```

Comment:
Er det naturlig å bruke chips med et valgfritt ikon inni?

<img width="412" alt="image" src="https://github.com/digdir/designsystem/assets/31471142/7c3b246d-849b-49f9-b8b9-8ab55429b17f">

Reply:
There should be no issues for our consumers to add their custom icon within the Chip today. The API allows the consumer of the component to extend it if needed. The Chip component also takes the ClassName attribute if overriding styling is needed.

```
<Chip.Group>
     <Chip onClick={myOnClickHandler}><CustomIconComponent/>My awesome Text</Chip>
     <Chip onClick={myOnClickHandler}><CustomIconComponent/>My awesome Text 2</Chip>
</Chip.Group>
```

Reply:
Oh, that's really nice 😊